backend/cmd/gateway/main.go

```diff
@@ -0,0 +1,106 @@
+// Package main implements main script to run gateway daemon.
```

Comment:
@juligasa consider renaming the binary to `mintter-gateway` or something more unique. If someone ever does `go install` with this binary package, it will be installed into theirs `$GOPATH/bin` which many people have added to their `$PATH`. Not a big deal at all, because no-one will probably do that. Still, a bit more ethical in my mind :)